client/components/TestRenderer/CommandResults/index.jsx

```diff
@@ -31,14 +37,41 @@ const CommandResults = ({
   });
   const tooltipID = useMemo(() => `untestable-tooltip-${++tooltipCount}`, []);
 
+  const hasUnresolvedAssertions = assertions.some(
+    assertion => assertion.passed === null
+  );
+
+  const hasConflictingOutput = isRerunReport && hasUnresolvedAssertions;
+
+  const errorMessage = hasConflictingOutput
+    ? `This output differs from output reported for ${historicalAtName} ${historicalAtVersion}`
+    : null;
+
+  const shouldShowHistoricalOutput =
+    hasConflictingOutput && historicalOutput !== null;
+
   return (
     <>
-      <h3>{header}</h3>
+      <h3>
+        {hasConflictingOutput && (
+          <FontAwesomeIcon
+            icon={faExclamationTriangle}
+            style={{ color: '#ce1b4c', marginRight: '8px' }}
+            aria-label="Conflicting Output"
+          />
```

Comment:
Oddly, your application of `aria-label` is *not* disabling the library's built-in application of `aria-hidden="true"` (bug report...?), so we need to do that explicitly.